docs/index.rst

```diff
@@ -99,6 +99,7 @@ Discover also `eXo UX Guidelines <http://exoplatform.github.io/ux-guidelines/>`_
    SSO
    WCM
    CMIS
+   Gamification
```

Comment:
unrelated. Should not be in this commit.

Reply:
I've started both documentations at the same time. My bad. i'll delete it.